Automattic/jetpack
# Newsletter Settings in JP: Display Email settings on Newsletter settings page

Closes https://github.com/Automattic/wp-calypso/issues/88308
Closes https://github.com/Automattic/wp-calypso/issues/88307
Closes https://github.com/Automattic/wp-calypso/issues/88306

## Proposed changes:

* Create EmailSettings component
* Display Email settings on Newsletter settings page
* Update API to support feature image and excerpt options

<img width="1089" alt="image" src="https://github.com/Automattic/jetpack/assets/3113712/de5749f8-0122-4e4c-833c-99ad946fdee0">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:

* Apply this patch to your JT site
* Go to the Newsletter settings page with the feature flag enabled: `{your-site}/wp-admin/admin.php?enable-email-settings=true&page=jetpack#/newsletter`
* The Email settings module should be displayed
* Perform tests on both Featured Image and Excerpt inputs, verify if the data is correctly synced with the settings on wpcom
* Remove the feature flag to make sure it is not displayed without the flag
